### PR TITLE
Feat: Added Avatar Components

### PR DESCRIPTION
**Pull request by shuntzhere**

### CODE REVIEW

File: avatar.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        <h1>Standard Avatars</h1>
+        <img class="avatar" src="/images/avatar.jpg" alt="Turtle" />
+        <img
+          class="avatar avatar-border"
+          src="./images/avatar.jpg"
+          alt="Turtle"
+        />
+      </div>
+
+      <br />
```
Comment (theprogrammerinyou) [inline avatar.html:23]: This <br /> tag is not required, as div is already a block element.

File: components/avatar.css [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  border-radius: 50%;
+  display: inline-block;
+  margin: 1rem;
+}
+
+.flex {
+  align-items: center;
+}
+
+h1 {
```
Comment (theprogrammerinyou) [inline components/avatar.css:28]: Have the typographies in typography.css file as base file and then if you want to overwrite you can write it here, but its better not to overwrite.

File: components/avatar.css [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  font-size: 1.5rem;
+}
+
+.avatar.sm {
```
Comment (theprogrammerinyou) [inline components/avatar.css:32]: Have these as well in root file.

**Review by theprogrammerinyou — CHANGES_REQUESTED** [review]
Please do the needful.

File: main.css [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+input,
+button,
+textarea,
+select {
+  font: inherit;
+}
+
+/* Base Classes */
+
+.flex {
```
Comment (Sameony) [inline main.css:47]: Redundant code, .flex redeclared, Line no.10

File: main.css [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+  width: 32px;
+  height: 32px;
+}
+
+.xl {
+  width: 40px;
+  height: 40px;
+}
+
+.xxl {
```
Comment (Sameony) [inline main.css:77]: Avoid Ambiguous class names

**Review by Sameony — COMMENTED** [review]
Suggesting Minor Adjustments

File: main.css (hunk 4 above)
Comment (shuntzhere) [reply main.css:47]: Yeah, it's updated now.

File: main.css (hunk 5 above)
Comment (shuntzhere) [reply main.css:77]: Yes, will change them.

File: components/avatar.css [hunk 6]
```diff
@@ -0,0 +1,43 @@
+*,
+*::after,
+*::before {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.avatars__container {
```
[the remark this answers is not in the document]
Comment (shuntzhere) [reply components/avatar.css:2]: Yes,  this is updated now.

File: components/avatar.css (hunk 2 above)
Comment (shuntzhere) [reply components/avatar.css]: Moved all to their specific files now.

File: components/avatar.css (hunk 3 above)
Comment (shuntzhere) [reply components/avatar.css]: Done.

**Review by Sameony — APPROVED** [review]

**Review by theprogrammerinyou — APPROVED** [review]

**Pull request merged**